html_app/components/FACSModelFactory.js

```diff
@@ -319,7 +313,17 @@ scb.components.FACSModelFactory = function scb_components_FACSModelFactory(model
                         }
                     },
 
-//                    tickFormatter: function (v, axis) {return "10^" + (Math.round( Math.log(v)/Math.LN10)).toString();}, //(Math.round( Math.log(v)/Math.LN10)).toString().sup();},
+                    tickFormatter: function (v, axis) {
+                        if (template.model.facs.scale && template.model.facs.scale.indexOf('pseudo') > -1) {
+                                return "10^"+Math.round(v/template.model.facs.ticks[0]);
+                        } else if (template.model.facs.scale && template.model.facs.scale.indexOf('log') > -1) {
```

Comment:
I'm curious: what are the possible values of template.model.facs.scale ? Are those documented anywhere?